project-base/storefront/cypress/e2e/transportAndPayment/transportSelect.cy.ts

```diff
@@ -18,6 +18,7 @@ describe('Transport select tests', () => {
         cy.addProductToCartForTest().then((cartUuid) => cy.storeCartUuidInLocalStorage(cartUuid));
         cy.visit(url.order.transportAndPayment);
 
+        cy.wait(200);
```

Comment:
For unknown reason Cypress is not willing to wait for this component to be loaded. On localhost this test will not fail but on CI it does. If you would know better solution I would highly appreciated.